Sources/SwiftCliCore/InstructionsGiver.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+public class InstructionsGiver {
+    var consoleWriter: ConsoleWriter
+
+    public init(consoleWriter: ConsoleWriter) {
+        self.consoleWriter = consoleWriter
+    }
+    public func giveInstructions() throws {
+        let allowedFileTypes = ".txt"
```

Comment:
I wonder if this should be a publicly available constant defined in `FileValidator.swfit` file.

Reply:
Done!